deps/libgit2.mk

```diff
@@ -61,6 +58,11 @@ CERTFILE=$(OPENSSL_DIR)/certs/ca.pem
 endif
 endif # Linux
 
+ifeq ($(OS),FreeBSD)
+OPENSSL_DIR=$(shell openssl version -d | cut -d '"' -f 2)
+CERTFILE=$(OPENSSL_DIR)/cert.pem
+endif # FreeBSD
+
```

Comment:
There is a block for Linux already: https://github.com/JuliaLang/julia/pull/22754/files#diff-a01ad0cac7aa5a648c5d00ec39bd9afcR44